Gemfile

```diff
@@ -40,17 +40,18 @@ group :development do
 end
 
 group :system_tests do
-  gem 'winrm',                              :require => false
-  if beaker_version = ENV['BEAKER_VERSION']
-    gem 'beaker', *location_for(beaker_version)
-  else
-    gem 'beaker', '>= 3.9.0', :require => false
-  end
-  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
-    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  gem 'pry'
```

Comment:
all those changes will be overwritten. I assume you used pry only once and it isn't needed always. Can you remove it from the PR? All Gemfiles are generated from this file: https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml. You can provide a PR to that repo if you like. Maybe it is a good idea to add the pry gems to the [development](https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L62) group.

Reply:
all pry stuffs are removed.